provider/cmd/pulumi-resource-buildkite/schema.json

```diff
@@ -30,7 +30,7 @@
         "go": {
             "generateExtraInputTypes": true,
             "generateResourceContainerTypes": true,
-            "importBasePath": "github.com/pulumiverse/pulumi-buildkite/sdk/go/buildkite"
+            "importBasePath": "github.com/pulumiverse/pulumi-buildkite/sdk/v2/go/buildkite"
```

Comment:
Assuming you will remain at major version 2 (`v2`) for a while, would it be possible to make this version number explicit in the `Installation & Configuration` docs here:

https://github.com/pulumiverse/pulumi-buildkite/blob/main/docs/installation-configuration.md?plain=1#L13

Reply:
fixed. github seems to have an issue with pushes to branches; it just doesn't update the PR changes or re-trigger the pipelines. you can see in the branch itself (which this PR references) the requested change: https://github.com/pulumiverse/pulumi-buildkite/blob/feat/deps/docs/installation-configuration.md?plain=1#L13